dashboards/datadog_audit_trail_overview/AuditTrailOverview.json

```diff
@@ -0,0 +1 @@
{"query":"$status_code $path $status $auth_method $user_agent $user $method -$exclude_status -$exclude_method"},"data_source":"audit","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"@http.url_details.path","sort":{"aggregation":"count","order":"desc"},"limit":1000}]}]}],"has_search_bar":"always"},"layout":{"x":7,"y":5,"width":3,"height":5}},{"id":8540485610202530,"definition":{"title":"$user_agent.value","title_size":"16","title_align":"left","type":"query_table","requests":[{"formulas":[{"alias":"count","conditional_formats":[],"limit":{"count":1000,"order":"desc"},"cell_display_mode":"bar","formula":"query1"}],"response_format":"scalar","queries":[{"search":{"query":"$status_code $path $status $auth_method $user_agent $user $method -$exclude_status -$exclude_method"},"data_source":"audit","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"@http.useragent","sort":{"aggregation":"count","order":"desc"},"limit":1000}]}]}],"has_search_bar":"never"},"layout":{"x":10,"y":5,"width":2,"height":5}}],"template_variables":[{"name":"status_code","default":"*","prefix":"@http.status_code","available_values":[]},{"name":"path","default":"*","prefix":"@http.url_details.path","available_values":[]},{"name":"status","default":"*","prefix":"status","available_values":[]},{"name":"auth_method","default":"API_AND_APP_KEY","prefix":"@auth_method","available_values":[]},{"name":"user_agent","default":"*","prefix":"@http.useragent","available_values":[]},{"name":"user","default":"*","prefix":"@usr.email","available_values":[]},{"name":"method","default":"*","prefix":"@http.method","available_values":[]},{"name":"exclude_status","default":"info","prefix":"status","available_values":[]},{"name":"exclude_method","default":"none","prefix":"@http.method","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"template_variable_presets":[{"name":"Bad Requests","template_variables":[{"name":"status_code","value":"400"},{"name":"exclude_status","value":"none"}]},{"name":"Conflict","template_variables":[{"name":"status_code","value":"409"},{"name":"exclude_status","value":"none"}]},{"name":"Forbidden","template_variables":[{"name":"status_code","value":"403"},{"name":"exclude_status","value":"none"}]},{"name":"Internal Server Error","template_variables":[{"name":"status_code","value":"500"},{"name":"exclude_status","value":"none"}]},{"name":"Not Found","template_variables":[{"name":"status_code","value":"404"},{"name":"exclude_status","value":"none"}]},{"name":"Quotas Reached","template_variables":[{"name":"status_code","value":"402"},{"name":"exclude_status","value":"none"}]},{"name":"Rate Limit Reached","template_variables":[{"name":"status_code","value":"429"},{"name":"exclude_status","value":"none"}]},{"name":"Unauthorized","template_variables":[{"name":"status_code","value":"401"},{"name":"exclude_status","value":"none"}]}],"reflow_type":"fixed","id":"r4f-knp-awx"}
```

Comment:
Can we remove this dashboard from this PR, please? (so we can focus on the second one, and review this one on the other PR)